stages/org.osbuild.ostree.deploy.container

```diff
@@ -109,6 +120,25 @@ def ostree_container_deploy(tree, inputs, osname, target_imgref, kopts):
             ostree.cli("container", "image", "deploy",
                        *extra_args, sysroot=tree, *kargs)
 
+            aleph_image_digest = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ .Digest }}"')
+            aleph_image_revision = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ index .Labels \"org.opencontainers.image.revision\" }}"')
+            aleph_image_source = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ index .Labels \"org.opencontainers.image.source\" }}"')
+            aleph_image_version = skopeo_cli("inspect", f"{image_source}", "--format", '"{{ index .Labels \"org.opencontainers.image.version\" }}"')
+
+            aleph_version_data = {
+                "osbuild-version": osbuild.__version__,
+                "image-name": image_name,
+                "image-digest": aleph_image_digest,
+                "image-revision": aleph_image_revision,
+                "image-source": aleph_image_source,
+                "image-version": aleph_image_version
+            }
```

Comment:
@cgwalters @jlebon Based on the discussion on the jira thread, this is what I've come up with. Feel free to let me know what should/shouldn't be here and I can make the changes. See the top comment of the PR for the sample output.